homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -1,31 +1,15 @@
-"""Support for Xiaomi Mi Air Purifier and Xiaomi Mi Air Humidifier."""
+"""Support for Xiaomi Mi Air Purifier."""
```

Comment:
```suggestion
"""Support for Xiaomi Mi Air Humidifier."""
```

Reply:
I don't understand, you want me to change to `Humidifier` in `fan.py` file? Maybe we should change this to `Support for Xiaomi Mi Fan and Xiaomi Mi Air Humidifier.`?

Reply:
I tricked myself. It's totally fine as it is.